sdk/dotnet/cmd/pulumi-language-dotnet/main.go

```diff
@@ -372,7 +372,7 @@ func DeterminePluginDependency(packageDir, packageName, packageVersion string) (
 	case os.IsNotExist(err):
 		break
 	case err != nil:
-		return nil, fmt.Errorf("Failed to read version file: %w", err)
```

Comment:
I've been wondering about this style myself for a while, @justinvp as you've been around long enough maybe there's a reason we we're doing capitalized error strings?

Reply:
I don't think there's a reason other than the author not knowing or not thinking about the conventions.

Reply:
I was very tangled when I made capitalization changes. I checked the test of the code and didn't see any impact, so I made changes. But worrying about whether it will affect other programs, I don't mind reverting this part of the changes. Thanks for your review.